weechat_otr_test/test_command_finish.py

```diff
@@ -21,23 +21,23 @@ def after_setup(self):
         weechat_otr.ACCOUNTS['nick@server'] = account
 
     def test_finish_buffer(self):
-        weechat_otr.command_cb(None, 'server_nick2_buffer', 'finish')
+        self.call_command_cb(None, 'server_nick2_buffer', 'finish')
```

Comment:
I think we should always call `to_str` when we call something that WeeChat calls in a test because WeeChat sends bytes.

Our strategy has been to convert bytes to unicode as soon as they come into our code and convert them back to bytes only on the way out if calling something that needs bytes.